pkg/provider/fortanix/fortanix.go

```diff
@@ -74,6 +73,28 @@ func (c *client) GetSecret(ctx context.Context, ref esv1beta1.ExternalSecretData
 	return utils.GetByteValue(value)
 }
 
+func (c *client) GetSecretMap(ctx context.Context, ref esv1beta1.ExternalSecretDataRemoteRef) (map[string][]byte, error) {
```

Comment:
This can be observed in many of the providers now sadly and none of them are using utils.JsonMarshal which escapes HTML. I'll allow it for now. But we'll definitely have to refactor that.

Reply:
Oh, I don't mind having a look and trying to refactor `utils.JsonMarshal`!

Reply:
Okay. :) If you feel up for it, the other providers also do something similar and create a map out of ref. :)

But if it's too much change, than, leave it. :)

Reply:
Correct me if I'm wrong, but as I understand `utils.JsonMarshal` should be used instead of raw `utils.GetByteValue`, right?

Then on GetSecretMap

```
        for k, v := range kv {
		secretData[k] = []byte(v)
	}
```

Should use `utils.JsonMarshal` instead of `[]byte(v)`

Reply:
Err, no, sorry. I didn't quite express myself right.

Instead of plain json.Marshal we use utils.JsonMarshal. And the whole `GetSecretMap` functionality is the one being repeated on other providers. You can ignore it. :D It was more of a not for myself really.